### PR TITLE
Ports ZAS from Baystation12 to replace Linda.

### PR DESCRIPTION
**Pull request by Kapu1178**
## About The Pull Request
Replaces the crusty old Linda with the objectively superior ZAS.

## Why It's Good For The Game
Linda's too slow, maximum speed.

## Changelog

:cl:
del: Linda atmospherics
del: Fusion
del: The Supermatter
del: The HFR
add: Zone Air System
add: Violent spacewind
add: Inflatable walls and doors
/:cl:

### CODE REVIEW

File: _maps/RandomRuins/IceRuins/icemoon_surface_asteroid.dmm [hunk 1]
```diff
@@ -12,20 +12,20 @@
 /obj/item/clothing/shoes/bronze,
 /obj/item/clothing/head/bronze,
 /turf/open/floor/bronze{
-	initial_gas_mix = "TEMP=2.7"
+	initial_gas = "TEMP=2.7"
```
Comment (JohnFulpWillard) [inline _maps/RandomRuins/IceRuins/icemoon_surface_asteroid.dmm:15]: why would you replace a var name with a worse one wtf

File: code/game/objects/items/latexballoon.dm [hunk 2]
```diff
@@ -13,7 +13,6 @@
 
 /obj/item/latexballon/Initialize(mapload)
```
Comment (JohnFulpWillard) [inline code/game/objects/items/latexballoon.dm:14]: this initialize isnt needed anymore

**Comment by miniusAreas** [comment]
DEAR GOD WHAT IS HAPPENING IN THERE.

**Comment by LemonInTheDark** [comment]
No

File: code/game/objects/items/grenades/atmos_grenades.dm [hunk 3]
```diff
@@ -93,7 +95,8 @@
 			continue
 		var/distance_from_center = max(get_dist(turf_loc, loc), 1)
 		var/turf/open/floor_loc = turf_loc
-		floor_loc.atmos_spawn_air("n2o=[n2o_gas_amount / distance_from_center];TEMP=273")
+		//floor_loc.atmos_spawn_air("n2o=[n2o_gas_amount / distance_from_center];TEMP=273")
```
Comment (JohnFulpWillard) [inline code/game/objects/items/grenades/atmos_grenades.dm:98]: this comment should be deleted if unused

File: code/game/objects/items/grenades/atmos_grenades.dm [hunk 4]
```diff
@@ -69,7 +69,9 @@
 			continue
 		var/distance_from_center = max(get_dist(turf_loc, loc), 1)
 		var/turf/open/floor_loc = turf_loc
-		floor_loc.atmos_spawn_air("n2=[n2_gas_amount / distance_from_center];o2=[o2_gas_amount / distance_from_center];TEMP=273")
+		//floor_loc.atmos_spawn_air("n2=[n2_gas_amount / distance_from_center];o2=[o2_gas_amount / distance_from_center];TEMP=273")
```
Comment (JohnFulpWillard) [inline code/game/objects/items/grenades/atmos_grenades.dm:72]: same here

**Comment by francinum** [comment]
> No

Counterpoint: Yes.

File: code/game/objects/items/powerfist.dm [hunk 5]
```diff
@@ -92,14 +92,14 @@
 		target.visible_message(span_danger("[user]'s powerfist lets out a dull thunk as [user.p_they()] punch[user.p_es()] [target.name]!"), \
 			span_userdanger("[user]'s punches you!"))
 		return
-	if(!molar_cmp_equals(gasused.total_moles(), gasperfist * fisto_setting))
+	/*if(!molar_cmp_equals(gasused.total_moles(), gasperfist * fisto_setting))
```
Comment (JohnFulpWillard) [inline code/game/objects/items/powerfist.dm:95]: awesome

File: code/game/objects/structures/aliens.dm [hunk 6]
```diff
@@ -63,22 +63,22 @@
...
 	max_integrity = 200
 	var/resintype = null
-	can_atmos_pass = ATMOS_PASS_DENSITY
+	can_atmos_pass = CANPASS_DENSITY
 
 
 /obj/structure/alien/resin/Initialize(mapload)
 	. = ..()
-	air_update_turf(TRUE, TRUE)
+	//air_update_turf(TRUE, TRUE)
```
Comment (JohnFulpWillard) [inline code/game/objects/structures/aliens.dm:71]: awesome

File: code/game/objects/structures.dm [hunk 7]
```diff
@@ -1,7 +1,7 @@
 /// Inert structures, such as girders, machine frames, and crates/lockers.
 /obj/structure
 	icon = 'icons/obj/structures.dmi'
-	pressure_resistance = 8
+	//pressure_resistance = 8
```
Comment (JohnFulpWillard) [inline code/game/objects/structures.dm:4]: awesome

File: code/game/objects/structures/aliens.dm [hunk 8]
```diff
@@ -63,22 +63,22 @@
...
 
 /obj/structure/alien/resin/Destroy()
-	air_update_turf(TRUE, FALSE)
+	//air_update_turf(TRUE, FALSE)
```
Comment (JohnFulpWillard) [inline code/game/objects/structures/aliens.dm:74]: awesome

**Review by JohnFulpWillard — COMMENTED** [review]
im just gonna stop

**Review by JohnFulpWillard — COMMENTED** [review]
blah i accidentally sent review twice

**Comment by miniusAreas** [comment]
> 

HOMOSUCK NERD RUINS JOKE

**Comment by LemonInTheDark** [comment]
Fucking arm

**Comment by LemonInTheDark** [comment]
stole my close meme 

> > No
> 
> Counterpoint: Yes.

**NO**

**Comment by francinum** [comment]
> > > No
> > 
> > 
> > Counterpoint: Yes.
> 
> **NO**

image

**Comment by Mothblocks** [comment]
Let's Make it a config

Dog smiling with grasshopper on its head

**Comment by Troontjelolo** [comment]
https://user-images.githubusercontent.com/56892466/165295782-c665f719-a994-4e05-9c05-4f5b4f76c363.mp4

**Pull request closed without merging**